### ISSUE TITLE
[21pt] Functionality to perform synthetic rating curve comparisons at all USGS gage locations

### ISSUE THREAD

**Issue by TrevorGrout-NOAA** [NOAA-OWP/cahaba#269]
Compare synthetic rating curves to observed rating curves associated with USGS gage locations. Calculate metrics such as bias, absolute difference, and etc.

Some specific steps that need to be taken are:
1) Ability to convert synthetic rating curve to elevation (NAVD88)
2) Crosswalk USGS gage location to a FIM HydroID so that correct rating curve can be selected.
3) Obtain USGS rating curve and convert to elevation (NAVD88)
4) Compare appropriate synthetic rating curve with USGS rating curve (calculate bias/mean absolute difference and etc).
5) Determine best methods for displaying and writing out datasets/plots

**Comment by BradfordBates-NOAA** [comment]
Let's start this during Sprint 10.

**Comment by TrevorGrout-NOAA** [comment]
Update: FIM v 3.0.14.0  completes the initial framework of the Sierra Test.  Remaining unchecked boxes (#286/#287) will be addressed as we further enhance this initial framework.

**Comment by BradfordBates-NOAA** [comment]
Agreed. I moved to Backlog until we prioritize #286 and #287, or add other tasks.
